lib/chef/provisioning/machine/unix_machine.rb

```diff
@@ -223,6 +223,10 @@ def detect_sh
 platform="aix"
 platform_version=`uname -v`
 machine="ppc"
+# Linux supporting /etc/os-release
```

Comment:
Not as far as I can see - is that script in Git somewhere that I can update too?